cmd/cmd.go

```diff
@@ -70,7 +70,7 @@ func initCommonFlags() ([]cli.Flag, error) {
 			&cli.StringFlag{
 				Name:     "hops",
 				Aliases:  []string{"H"},
-				Usage:    "Path to dir containing *.hops configs (or path to single *.hops file)",
+				Usage:    "Path to dir containing subdirectories of *.hops configs (and additional files)",
```

Comment:
I don't think this would be a meaningful message to anyone that didn't already understand the mechanisms at play (and therefore didn't need the message).

I'd go with "Path to dir containing hiphops automations" since we'll be using the "automations" language in UIs and docs from now on.

Reply:
(This one is a must fix btw)